examples/webgl_loader_gltf_transmission.html

```diff
@@ -51,15 +52,21 @@
 						texture.dispose();
 						pmremGenerator.dispose();
 
-						render();
-
 						// model
 
-						const loader = new GLTFLoader().setPath( 'models/gltf/' );
-						loader.load( 'TransmissionTest.glb', function ( gltf ) {
+						const loader = new GLTFLoader().setPath( 'models/gltf/IridescentDishWithOlives/glTF/' );
+						loader.load( 'IridescentDishWithOlives.gltf', function ( gltf ) {
+
+							mixer = new THREE.AnimationMixer( gltf.scene );
+
+							for ( let i = 0; i < gltf.animations.length; i ++ ) {
+
+								mixer.clipAction( gltf.animations[ i ] ).play();
+
+							}
```

Comment:
This example only has 1 animation, and I'd rather not encourage people to play all animations. Playing just the first is usually a better default, and encourages exporters to meaningfully group their animations. Do you mind changing this to:

```
mixer.clipAction( gltf.animations[ 0 ] ).play();
```

Reply:
Sounds good to me. Updated, thanks!